src/logics/Logic.h

```diff
@@ -186,6 +186,10 @@ class Logic {
     SymRef declareFun_Commutative_NoScoping_LeftAssoc(std::string const & s, SRef rsort, vec<SRef> const & args) {
         return declareFun(s, rsort, args, SymConf::CommutativeNoScopingLeftAssoc);
     }
+    SymRef declareFun_Multiplication_Duplicate(std::string const & s, SRef rsort, vec<SRef> const & args) {
```

Comment:
This should not be public.

Reply:
It should probably not even be here, but in `ArithLogic`.

Reply:
Moved to ArithLogic, isn't public anymore